create_compendiums.py

```diff
@@ -73,24 +163,54 @@ def create_category_compendiums():
     :return: list of output paths.
 
     """
-    categories = ['Items', 'Character', 'Spells', 'Bestiary']
+    categories = ['Items', 'Character', 'Spells', 'Bestiary', 'Unearthed Arcana']
     output_paths = []
     for category in categories:
         filenames = glob('%s/*.xml' % category)
         output_path = COMPENDIUM.format(category=category)
-        output_paths.append(output_path)
+
+        """build UA compendium, but exclude from Full"""
+        if output_path != 'Unearthed Arcana':
+            output_paths.append(output_path)
```

Comment:
bug here. it was supposed to test `category != 'Unearthed Arcana'` to keep that out of Full